src/Congo.WebApi/Startup.cs

```diff
@@ -24,8 +28,30 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddAppServices(Configuration);
-
             services.AddControllers();
+
+            services.AddCors(options =>
+            {
+                options.AddPolicy(_CORS_DEV, policy =>
+                            policy.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader());
+
+                options.AddPolicy(_CORS_PROD, policy =>
+                {
+                    // TODO: Multiple Origins from env vars
+                    string ALLOWED_ORIGIN = Environment.GetEnvironmentVariable("ALLOWED_ORIGIN");
+
+                    if (!string.IsNullOrWhiteSpace(ALLOWED_ORIGIN))
+                    {
+                        //Debug.WriteLine("ALLOWED_ORIGIN = " + ALLOWED_ORIGIN);
+                        policy.WithOrigins(ALLOWED_ORIGIN).AllowAnyMethod().AllowAnyHeader();
+                    }
+                    else if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") != "Development")
+                    {
+                        // by default, this exception is thrown even in dev env without the above check
```

Comment:
Try to not comment what code does, but why. Anyone who understands C# knows this will throw in development without the above check, but it might not be instantly obvious why we don't want that.

Reply:
Got it. But with English being my 2nd language, I really can't think of anything else. 

There aren't any reasons except for the exception is thrown even in dev, while the code is expected get executed in production. Maybe, More like a workaround.

Reply:
How about...
```suggestion
                        // Don't throw any exceptions in development, as this policy should only be used in production
```

Reply:
Looks good to me. 

But I can say, My brain never felt this confused over something that this basic. 😅